.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  schedule:
+    - cron: '0 0 1,15 *'
+  workflow_dispatch:
+
+name: Launch Scala Steward
+
+jobs:
+  scala-steward:
+    runs-on: ubuntu-latest
+    name: Launch Scala Steward
+    steps:
+      - name: Launch Scala Steward
+        uses: scala-steward-org/scala-steward-action@v2
+        with:
+          github-token: ${{ secrets.SCALAMETA_BOT_GH_TOKEN }}
+          author-email: scalameta@gmail.com
+          author-name: Scalameta Bot
```

Comment:
I'm not 100% sure this will correctly tie it to the scalameta-bot account, but I _think_ it might.

Reply:
I think this needs to match the public email and name of the bot, but I don't think the bot has either.